recipes/configure-ca-webapp-gunicorn.rb

```diff
@@ -0,0 +1,29 @@
+# Cookbook Name:: mh-opsworks-recipes
+# Recipe:: configure-ca-webapp-gunicorn
+
+::Chef::Recipe.send(:include, MhOpsworksRecipes::RecipeHelpers)
+
+ca_webapp_info = node.fetch(:ca_webapp, {})
+app_name = ca_webapp_info.fetch(:ca_webapp_name, "webapp")
+
+execute "install gunicorn" do
+  command "source /home/web/sites/#{app_name}/venv/bin/activate && pip install gunicorn"
+  user "web"
```

Comment:
Could we name this user something more appropriate for this tooling? maybe "capture_agent_manager?" It'd be nice to be able to understand from `ps` (and similar tools) exactly what a process is for.